### ISSUE TITLE
app thread: timeline not working - I cant see nothing about timeline

### ISSUE THREAD

**Issue by PhilippeAccorsi** [tracim/tracim#1056]
At present:

When I change status or archive/delete thread nothing is show in app.

Expected

Have same functionnality like we have in timeline.

**Comment by lebouquetin** [comment]
@Skylsmoi the implementation of the timeline must be unique - use same part of frontend code for all apps (htmldocument, file and thread)

**Comment by Skylsmoi** [comment]
@lebouquetin everything comes from frontend_lib

**Comment by Skylsmoi** [comment]
the app thread is just a timeline in full width

**Comment by lebouquetin** [comment]
@Skylsmoi 

- if I change the label of a html document, a new revision appears in the timeline
- if I change the label of a thread, nothing is visible

=> so, No the app thread is not "just a timeline in full width". Maybe the pb is in the backend, but there is a problem somewhere.

By the way, the problem I describes in current comment is already explained in the issue description. I do not expect a response like "this is exactly the same thing" while it's not.

**Comment by lebouquetin** [comment]
@Skylsmoi ok. sorry for the misunderstanding. But... I do not expect you to develop "again" the management of the status change / etc". I expect it to be in frontendlib because it is exactly the same behavior for all timelines. It's not reuse of "visual components", it's also reuse of "logic for filling the timeline and interacting with it"

**Comment by Skylsmoi** [comment]
@lebouquetin @PhilippeAccorsi 
  
I need the exhaustive list of revision type that I should display in thread timeline
Here is the list from frontend_lib (built from backend list)
```
[{
    id: 'archiving',
    faIcon: 'archive',
    label: i18n.t('Item archived')
  }, {
    id: 'content-comment',
    faIcon: 'comment-o',
    label: i18n.t('Comment')
  }, {
    id: 'creation',
    faIcon: 'magic',
    label: i18n.t('Item created')
  }, {
    id: 'deletion',
    faIcon: 'trash',
    label: i18n.t('Item deleted')
  }, {
    id: 'edition',
    faIcon: 'edit',
    label: i18n.t('New revision')
  }, {
    id: 'revision',
    faIcon: 'history',
    label: i18n.t('New revision')
  }, {
    id: 'status-update',
    faIcon: 'random',
    label: i18n.t('New status')
  }, {
    id: 'unarchiving',
    faIcon: 'file-archive-o',
    label: i18n.t('Item unarchived')
  }, {
    id: 'undeletion',
    faIcon: 'trash-o',
    label: i18n.t('Item restored')
  }, {
    id: 'move',
    faIcon: 'arrows',
    label: i18n.t('Item moved')
  }, {
    id: 'copy',
    faIcon: 'files-o',
    label: i18n.t('Item copied')
  }]
```

Right now, theses are the types I keep in app thread:
`['comment', 'archiving', 'deletion', 'status-update', 'unarchiving', 'undeletion', 'move', 'edition']`

**Comment by Skylsmoi** [comment]
Since I have only removed the "creation" type, maybe we can just keep everything in the thread app

**Comment by lebouquetin** [comment]
@Skylsmoi yes, keep everything. The only specificity which would be interesting to keep is the label associated to timeline items.

Eg `creation` label would be `Creation` for a file or a document and `Start` for a thread.
